### ISSUE TITLE
Make Custom Full Layout should create a scene based on the layers in the layout

### ISSUE THREAD

**Issue by nlupugla** [dialogic-godot/dialogic#1985]
**Is your feature request related to a problem? Please describe.**
When I make a full custom layout, Dialogic starts me with a nearly-empty scene that just contains a DialogicLayoutBase node. This is frustrating if you want a custom layout that is pretty close to one of the presets except for a few modifications, as you essentially have to manually rebuild the preset. I also find it easier to edit my layout from Godot's scene editor compared to Dialogic's Style UI as the scene editor gives me instant feedback compared to entering a value, hitting "Test Style", then waiting for the scene to load.

If the user wanted a blank slate, they probably would have selected the "+ -> Custom Style" option from the Styles menu. 

**Describe the solution you'd like**
When the user selects "Make Custom -> Full Layout" from the Layers menu, the created scene should essentially match the scene you would create if you had hit the "Test Style" button. 

**Describe alternatives you've considered**
The created scene could instead match the default scene for whatever premade style the user had previously selected, but I think this would be both harder to implement and less useful overall.

Edit:

Another alternative could be to add a "Save Style" button next to the "Test Style" button. The "Save Style" button would create a .tscn file describing the scene that would be created when hitting "Test Style".

**Additional context**

**Comment by Jowan-Spooner** [comment]
Hmm... I'm not sure if I'm understanding you correctly, but if I use "MakeCustom>FullLayout" I already get a scene that is exactly the same as what is used when pressing TestTimeline. (It might not look like it, but it is).

I assume the problem here is that using this button will keep all the options as options and they are not live-applied in editor, as that can ruin them. 

The problem is that the options are hard to remove automatically from the scene without messing it up.
I think if you don't want to keep the options, going for a static and simpler custom scene (made from scratch) would be the best way anyways.

You can try to add an alternative button (I assume it will be hard), but the current option of realizing the scene has to stay.

**Comment by nlupugla** [comment]
Ah, I think I understand what's going on here. It is related to getting parse errors on account some GDScript settings: https://github.com/dialogic-godot/dialogic/pull/1975

Here's is what happens when I GDScript settings are all default and I believe is the intended behavior.

Starting from the premade visual novel scene, I click Make Custom -> Full Layout

<img width="1120" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/974d2f92-1425-4569-9a4f-94e5716bd1f6">

The layer window now looks like so

<img width="145" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/c0148d3a-15f8-4e94-bbef-f8020cbf1f57">

and inspecting the scene reveals a full custom layout ready for edits:

<img width="1120" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/b057d241-b6a4-4bd6-ab1a-224276ed7d30">

However, this is what happens when I go through the same process, but with slightly stricter GDScript settings:

<img width="452" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/9be9e611-92d8-4856-80ef-12ff1af0714e">

I hit Make Custom -> Full Layout and the Layers panel displays

<img width="181" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/78de88e7-3d2c-4e23-80d9-a74d9383f5fa">

Inspecting the Layout scene reveals a blank scene:

<img width="1120" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/45b83b37-6015-4da4-8aa0-889dfff42005">

For completeness, this is the error log:

<img width="840" alt="image" src="https://github.com/dialogic-godot/dialogic/assets/12731832/c65f43b6-a930-4e45-bc6c-6f1fa7785775">

TLDR: the intended behavior seems to be essentially what I described in this proposal, which is great! On the other hand, I wasn't getting the intended behavior because of GDScript settings that are probably fairly common (although not default).
